training_record/views.py

```diff
@@ -47,7 +48,7 @@ def reviewed(self, request):
 
 # pylint: disable=no-self-use
 class RecordActionViewSet(viewsets.ViewSet):
-    '''Define actions for admins to batch submit Records of campus event.'''
+    '''Define actions for admins to manipulate Record objects.'''
```

Comment:
I think we can describe this viewset as `Create extra actions for Record objects.`.